app/src/main/java/cmp/sem/team8/smarlecture/quickstatistics/QuickStatisticsActivity.java

```diff
@@ -22,53 +20,52 @@
 import com.google.firebase.database.FirebaseDatabase;
 import com.google.firebase.database.ValueEventListener;
 
-import java.util.HashMap;
-import java.util.Map;
-
 import cmp.sem.team8.smarlecture.R;
 import cmp.sem.team8.smarlecture.auth.LoginActivity;
 import cmp.sem.team8.smarlecture.joinsession.JoinedSession;
+import cmp.sem.team8.smarlecture.profile.ProfileActivity;
 import cmp.sem.team8.smarlecture.session.SessionActivity;
 
 public class QuickStatisticsActivity extends AppCompatActivity {
 
-private Button BeginSession;
-private Button JoinSession;
-private Context context;
-private DatabaseReference ref;
-
+    private Button BeginSession;
+    private Button JoinSession;
+    private Context context;
+    private DatabaseReference ref;
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_quick_statistics);
 
-        context=this;
+        context = this;
 
         Intent intent = new Intent(this, LoginActivity.class);
         startActivity(intent);
 
-        ref=FirebaseDatabase.getInstance().getReference();
 
-        BeginSession=(Button)findViewById(R.id.button3);
+        // TODO: This was conflict, now the same buttons has 2 diff listeners. Update it
+        ref = FirebaseDatabase.getInstance().getReference();
```

Comment:
I think this is should be removed, this is the firebase ?!

Reply:
ok